eng/basic_tag_matrix.json

```diff
@@ -0,0 +1,7 @@
+{
+  "matrix": {
+    "Tag": [
+      "js/openai/openai_f96a922c3c"
```

Comment:
This exists purely as an example. It always gets replaced on an invocation of the batch build.